CREATE TABLE `MUTATIONS_CLINVAR`(
  `ROWID` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `FILEID` INT,
  `CHROM` VARCHAR(100) COMMENT 'Chromosome',
  `POS` INT COMMENT 'Position',
  `ID`  INT COMMENT 'Identifier',
  `REF` VARCHAR(100) COMMENT 'Reference base(s)',
  `ALT` VARCHAR(100) COMMENT 'Alternate base(s)',
  `QUAL` FLOAT COMMENT 'Quality',
  `FILTER` VARCHAR(100) COMMENT 'Filter status',
  `info_RS` INT COMMENT 'dbSNP ID (i.e. rs number)',
  `info_RSPOS` INT COMMENT 'Chr position reported in dbSNP',
  `info_RV` INT COMMENT 'RS orientation is reversed',
  `info_VP` VARCHAR(100) COMMENT 'Variation Property.  Documentation is at ftp://ftp.ncbi.nlm.nih.gov/snp/specs/dbSNP_BitField_latest.pdf',
  `info_GENEINFO` VARCHAR(100) COMMENT 'Pairs each of gene symbol:gene id.  The gene symbol and id are delimited by a colon (:) and each pair is delimited by a vertical bar (|)',
  `info_dbSNPBuildID` INT COMMENT 'First dbSNP Build for RS',
  `info_SAO` INT COMMENT 'Variant Allele Origin: 0 - unspecified 1 - Germline 2 - Somatic 3 - Both',
  `info_SSR` INT COMMENT 'Variant Suspect Reason Codes (may be more than one value added together) 0 - unspecified 1 - Paralog 2 - byEST 4 - oldAlign 8 - Para_EST 16 - 1kg_failed 1024 - other',
  `info_WGT` INT COMMENT 'Weight 00 - unmapped 1 - weight 1 2 - weight 2 3 - weight 3 or more',
  `info_VC` VARCHAR(100) COMMENT 'Variation Class',
  `info_PM` INT COMMENT 'Variant is Precious(ClinicalPubmed Cited)',
  `info_TPA` INT COMMENT 'Provisional Third Party Annotation(TPA) (currently rs from PHARMGKB who will give phenotype data)',
  `info_PMC` INT COMMENT 'Links exist to PubMed Central article',
  `info_S3D` INT COMMENT 'Has 3D structure - SNP3D table',
  `info_SLO` INT COMMENT 'Has SubmitterLinkOut - From SNP-SubSNP-Batch.link_out',
  `info_NSF` INT COMMENT 'Has non-synonymous frameshift A coding region variation where one allele in the set changes all downstream amino acids. FxnClass = 44',
  `info_NSM` INT COMMENT 'Has non-synonymous missense A coding region variation where one allele in the set changes protein peptide. FxnClass = 42',
  `info_NSN` INT COMMENT 'Has non-synonymous nonsense A coding region variation where one allele in the set changes to STOP codon (TER). FxnClass = 41',
  `info_REF` INT COMMENT 'Has reference A coding region variation where one allele in the set is identical to the reference sequence. FxnCode = 8',
  `info_SYN` INT COMMENT 'Has synonymous A coding region variation where one allele in the set does not change the encoded amino acid. FxnCode = 3',
  `info_U3` INT COMMENT 'In 3 UTR Location is in an untranslated region (UTR). FxnCode = 53',
  `info_U5` INT COMMENT 'In 5 UTR Location is in an untranslated region (UTR). FxnCode = 55',
  `info_ASS` INT COMMENT 'In acceptor splice site FxnCode = 73',
  `info_DSS` INT COMMENT 'In donor splice-site FxnCode = 75',
  `info_INT` INT COMMENT 'In Intron FxnCode = 6',
  `info_R3` INT COMMENT 'In 3 gene region FxnCode = 13',
  `info_R5` INT COMMENT 'In 5 gene region FxnCode = 15',
  `info_OTH` INT COMMENT 'Has other variant with exactly the same set of mapped positions on NCBI refernce assembly.',
  `info_CFL` INT COMMENT 'Has Assembly conflict. This is for weight 1 and 2 variant that maps to different chromosomes on different assemblies.',
  `info_ASP` INT COMMENT 'Is Assembly specific. This is set if the variant only maps to one assembly',
  `info_MUT` INT COMMENT 'Is mutation (journal citation explicit fact): a low frequency variation that is cited in journal and other reputable sources',
  `info_VLD` INT COMMENT 'Is Validated.  This bit is set if the variant has 2+ minor allele count based on frequency or genotype data.',
  `info_G5A` INT COMMENT '5% minor allele frequency in each and all populations',
  `info_G5` INT COMMENT '5% minor allele frequency in 1+ populations',
  `info_HD` INT COMMENT 'Marker is on high density genotyping kit (50K density or greater).  The variant may have phenotype associations present in dbGaP.',
  `info_GNO` INT COMMENT 'Genotypes available. The variant has individual genotype (in SubInd table).',
  `info_KGPhase1` INT COMMENT '1000 Genome phase 1 (incl. June Interim phase 1)',
  `info_KGPhase3` INT COMMENT '1000 Genome phase 3',
  `info_CDA` INT COMMENT 'Variation is interrogated in a clinical diagnostic assay',
  `info_LSD` INT COMMENT 'Submitted from a locus-specific database',
  `info_MTP` INT COMMENT 'Microattribution/third-party annotation(TPA:GWASPAGE)',
  `info_OM` INT COMMENT 'Has OMIM/OMIA',
  `info_NOC` INT COMMENT 'Contig allele not present in variant allele list. The reference sequence allele at the mapped position is not present in the variant allele list adjusted for orientation.',
  `info_WTD` INT COMMENT 'Is Withdrawn by submitter If one member ss is withdrawn by submitter then this bit is set.  If all member ss are withdrawn then the rs is deleted to SNPHistory',
  `info_NOV` INT COMMENT 'Rs cluster has non-overlapping allele sets. True when rs set has more than 2 alleles from different submissions and these sets share no alleles in common.',
  `info_CAF` VARCHAR(100) COMMENT 'An ordered comma delimited list of allele frequencies based on 1000Genomes starting with the reference allele followed by alternate alleles as ordered in the ALT column. Where a 1000Genomes alternate allele is not in the dbSNPs alternate allele set the allele is added to the ALT column.  The minor allele is the second largest value in the list and was previuosly reported in VCF as the GMAF.  This is the GMAF reported on the RefSNP and EntrezSNP pages and VariationReporter',
  `info_COMMON` INT COMMENT 'RS is a common SNP.  A common SNP is one that has at least one 1000Genomes population with a minor allele of frequency = 1% and for which 2 or more founders contribute to that minor allele frequency.',
  `info_CLNHGVS` VARCHAR(100) COMMENT 'Variant names from HGVS.    The order of these variants corresponds to the order of the info in the other clinical  INFO tags.',
  `info_CLNALLE` INT COMMENT 'Variant alleles from REF or ALT columns.  0 is REF 1 is the first ALT allele etc.  This is used to match alleles with other corresponding clinical (CLN) INFO tags.  A value of -1 indicates that no allele was found to match a corresponding HGVS allele name.',
  `info_CLNSRC` VARCHAR(100) COMMENT 'Variant Clinical Chanels',
  `info_CLNORIGIN` VARCHAR(100) COMMENT 'Allele Origin. One or more of the following values may be added: 0 - unknown; 1 - germline; 2 - somatic; 4 - inherited; 8 - paternal; 16 - maternal; 32 - de-novo; 64 - biparental; 128 - uniparental; 256 - not-tested; 512 - tested-inconclusive; 1073741824 - other',
  `info_CLNSRCID` VARCHAR(100) COMMENT 'Variant Clinical Channel IDs',
  `info_CLNSIG` VARCHAR(100) COMMENT 'Variant Clinical Significance 0 - Uncertain significance 1 - not provided 2 - Benign 3 - Likely benign 4 - Likely pathogenic 5 - Pathogenic 6 - drug response 7 - histocompatibility 255 - other',
  `info_CLNDSDB` VARCHAR(100) COMMENT 'Variant disease database name',
  `info_CLNDSDBID` VARCHAR(100) COMMENT 'Variant disease database ID',
  `info_CLNDBN` VARCHAR(100) COMMENT 'Variant disease name',
  `info_CLNREVSTAT` VARCHAR(100) COMMENT 'ClinVar Review Status mult - Classified by multiple submitters single - Classified by single submitter not - Not classified by submitter exp - Reviewed by expert panel prof - Reviewed by professional society',
  `info_CLNACC` VARCHAR(100) COMMENT 'Variant Accession and Versions'
);
alter table MUTATIONS_CLINVAR add key (`ID`);

