CREATE TABLE `MUTATIONS`(
  `ROWID` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `FILEID` INT,
  `CHROM` VARCHAR(1000) COMMENT 'Chromosome',
  `POS` INT COMMENT 'Position',
  `ID`  VARCHAR(1000) COMMENT 'Identifier',
  `REF` VARCHAR(1000) COMMENT 'Reference base(s)',
  `ALT` VARCHAR(1000) COMMENT 'Alternate base(s)',
  `QUAL` FLOAT COMMENT 'Quality',
  `FILTER` VARCHAR(1000) COMMENT 'Filter status',
  `info_NS` INT COMMENT 'Number of Samples With Data',
  `info_AN` INT COMMENT 'Total number of alleles in called genotypes',
  `info_AC` INT COMMENT 'Allele count in genotypes, for each ALT allele',
  `info_CGA_XR` VARCHAR(1000) COMMENT 'Per-ALT external database reference (dbSNP, COSMIC, etc)',
  `info_CGA_FI` VARCHAR(1000) COMMENT 'Functional impact annotation',
  `info_CGA_PFAM` VARCHAR(1000) COMMENT 'PFAM Domain',
  `info_CGA_MIRB` VARCHAR(1000) COMMENT 'miRBaseId',
  `info_CGA_RPT` VARCHAR(1000) COMMENT 'repeatMasker overlap information',
  `info_CGA_SDO` INT COMMENT 'Number of distinct segmental duplications that overlap this locus',
  `info_END` INT COMMENT 'End position of the variant described in this record',
  `info_CGA_WINEND` INT COMMENT 'End of coverage window',
  `info_CGA_BF` FLOAT COMMENT 'Frequency in baseline',
  `info_CGA_MEDEL` VARCHAR(1000) COMMENT 'Consistent with deletion of mobile element; type,chromosome,start,end',
  `info_MATEID` VARCHAR(1000) COMMENT 'ID of mate breakend',
  `info_SVTYPE` VARCHAR(1000) COMMENT 'Type of structural variant',
  `info_CGA_BNDG` VARCHAR(1000) COMMENT 'Transcript name and strand of genes containing breakend',
  `info_CGA_BNDGO` VARCHAR(1000) COMMENT 'Transcript name and strand of genes containing mate breakend',
  `info_CIPOS` INT COMMENT 'Confidence interval around POS for imprecise variants',
  `info_IMPRECISE` BIT COMMENT 'Imprecise structural variation',
  `info_MEINFO` VARCHAR(1000) COMMENT 'Mobile element info of the form NAME,START,END,POLARITY',
  `info_SVLEN` INT COMMENT 'Difference in length between REF and ALT alleles',
  `info_ANN`  VARCHAR(1000) COMMENT 'snpEff Annotation',
  `info_LOF`  VARCHAR(1000) COMMENT 'snpEff LOF',
  `info_NMD`   VARCHAR(1000) COMMENT 'snpEff NMD',
  `format_GT` VARCHAR(1000) COMMENT 'Genotype',
  `format_PS` INT COMMENT 'Phase Set',
  `format_SS` VARCHAR(1000) COMMENT 'Somatic Status: Germline, Somatic, LOH, or . (Unknown)',
  `format_FT` VARCHAR(1000) COMMENT 'Genotype filters',
  `format_GQ` INT COMMENT 'Genotype Quality',
  `format_HQ` INT COMMENT 'Haplotype Quality',
  `format_EHQ` INT COMMENT 'Haplotype Quality, Equal Allele Fraction Assumption',
  `format_CGA_CEHQ` INT COMMENT 'Calibrated Haplotype Quality, Equal Allele Fraction Assumption',
  `format_GL` INT COMMENT 'Genotype Likelihood',
  `format_CGA_CEGL` INT COMMENT 'Calibrated Genotype Likelihood, Equal Allele Fraction Asssumption',
  `format_DP` INT COMMENT 'Total Read Depth',
  `format_AD` INT COMMENT 'Allelic depths (number of reads in each observed allele)',
  `format_CGA_RDP` INT COMMENT 'Number of reads observed supporting the reference allele',
  `format_CGA_GP` FLOAT COMMENT 'Depth of coverage for 2k window GC normalized to mean',
  `format_CGA_NP` FLOAT COMMENT 'Coverage for 2k window, GC-corrected and normalized relative to copy-number-corrected multi-sample baseline',
  `format_CGA_CL` FLOAT COMMENT 'Nondiploid-model called level',
  `format_CGA_LS` INT COMMENT 'Nondiploid-model called level score',
  `format_CGA_CP` INT COMMENT 'Diploid-model called ploidy',
  `format_CGA_PS` INT COMMENT 'Diploid-model called ploidy score',
  `format_CGA_CT` VARCHAR(1000) COMMENT 'Diploid-model CNV type',
  `format_CGA_TS` INT COMMENT 'Diploid-model CNV type score',
  `format_CGA_BNDMPC` INT COMMENT 'Mate pair count supporting breakend',
  `format_CGA_BNDPOS` INT COMMENT 'Breakend position',
  `format_CGA_BNDDEF` VARCHAR(1000) COMMENT 'Breakend definition',
  `format_CGA_BNDP` VARCHAR(1000) COMMENT 'Precision of breakend',
  `format_CGA_IS` FLOAT COMMENT 'MEI InsertionScore: confidence in occurrence of an insertion',
  `format_CGA_IDC` FLOAT COMMENT 'MEI InsertionDnbCount: count of paired ends supporting insertion',
  `format_CGA_IDCL` FLOAT COMMENT 'MEI InsertionLeftDnbCount: count of paired ends supporting insertion on 5 end of insertion point',
  `format_CGA_IDCR` FLOAT COMMENT 'MEI InsertionRightDnbCount: count of paired ends supporting insertion on 3 end of insertion point',
  `format_CGA_RDC` INT COMMENT 'MEI ReferenceDnbCount: count of paired ends supporting reference allele',
  `format_CGA_NBET` VARCHAR(1000) COMMENT 'MEI NextBestElementType: (sub)type of second-most-likely inserted mobile element',
  `format_CGA_ETS` FLOAT COMMENT 'MEI ElementTypeScore: confidence that insertion is of type indicated by CGA_ET/ElementType',
  `format_CGA_KES` FLOAT COMMENT 'MEI KnownEventSensitivityForInsertionScore: fraction of known MEI insertion polymorphisms called for this sample with CGA_IS at least as high as for the current call'
);
